server/boot/user.js

```diff
@@ -223,7 +223,7 @@ module.exports = function(app) {
     (req, res) => res.redirect(req.url.replace('full-stack', 'back-end'))
   );
 
-  router.get('/:username', showUserProfile);
+  // router.get('/:username', showUserProfile);
```

Comment:
remove unwanted comments, also this particular change is blocking a feature.

Reply:
temporary